.github/workflows/rsc/dockerfile_lint_rules.yaml

```diff
@@ -0,0 +1,213 @@
+---
```

Comment:
You should rename this file to start with a "." i.e. `.dockerfile_lint_rules.yaml` otherwise statick-tooling will discover this as an actual Dockerfile and try to run the linters on it.
this is because the best way to differentiate Dockerfiles from other files is the string prefix "Dockerfile" at the beginning of the filename.

Reply:
this should be resolved once this PR to ignore yaml files is merged and released: https://github.com/sscpac/statick-tooling/pull/12